<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>2&nbsp; Literature review – Second year progress review</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./preprint.html" rel="next">
<link href="./summary.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./Lit_review.html"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Literature review</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">Second year progress review</a> 
        <div class="sidebar-tools-main">
    <a href="https://github.com/Harbour-N/Second-year-report" title="Source Code" class="quarto-navigation-tool px-1" aria-label="Source Code"><i class="bi bi-github"></i></a>
    <div class="dropdown">
      <a href="" title="Download" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" role="link" aria-label="Download"><i class="bi bi-download"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="./Second-year-progress-review.pdf">
              <i class="bi bi-file-pdf pe-1"></i>
            Download PDF
            </a>
          </li>
          <li>
            <a class="dropdown-item sidebar-tools-main-item" href="./Second-year-progress-review.docx">
              <i class="bi bi-file-word pe-1"></i>
            Download Docx
            </a>
          </li>
      </ul>
    </div>
</div>
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Preface</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./summary.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Summary</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./Lit_review.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Literature review</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./preprint.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Virtual Clinical Trials of BMP4 Differentiation Therapy: Digital Twins to Aid Glioblastoma Trial Design</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">References</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#sec-introduction-lit" id="toc-sec-introduction-lit" class="nav-link active" data-scroll-target="#sec-introduction-lit"><span class="header-section-number">2.1</span> Introduction</a></li>
  <li><a href="#sec-stem-cells-in-cancer" id="toc-sec-stem-cells-in-cancer" class="nav-link" data-scroll-target="#sec-stem-cells-in-cancer"><span class="header-section-number">2.2</span> Stem cells in cancer</a></li>
  <li><a href="#sec-cancer-stem-cells-and-treatment-resistance" id="toc-sec-cancer-stem-cells-and-treatment-resistance" class="nav-link" data-scroll-target="#sec-cancer-stem-cells-and-treatment-resistance"><span class="header-section-number">2.3</span> Cancer stem cells and treatment resistance</a></li>
  <li><a href="#sec-mathematical-models-of-cancer-stem-cell-dynamics" id="toc-sec-mathematical-models-of-cancer-stem-cell-dynamics" class="nav-link" data-scroll-target="#sec-mathematical-models-of-cancer-stem-cell-dynamics"><span class="header-section-number">2.4</span> Mathematical models of cancer stem cell dynamics</a>
  <ul class="collapse">
  <li><a href="#sec-agent-based-model" id="toc-sec-agent-based-model" class="nav-link" data-scroll-target="#sec-agent-based-model"><span class="header-section-number">2.4.1</span> Agent-based models</a></li>
  <li><a href="#sec-integro-differential-model" id="toc-sec-integro-differential-model" class="nav-link" data-scroll-target="#sec-integro-differential-model"><span class="header-section-number">2.4.2</span> Integro-differential model</a></li>
  <li><a href="#sec-ode-model-reduction" id="toc-sec-ode-model-reduction" class="nav-link" data-scroll-target="#sec-ode-model-reduction"><span class="header-section-number">2.4.3</span> ODE model reduction</a></li>
  <li><a href="#sec-stochastic-model" id="toc-sec-stochastic-model" class="nav-link" data-scroll-target="#sec-stochastic-model"><span class="header-section-number">2.4.4</span> Stochastic model of CSC dynamics</a></li>
  <li><a href="#sec-multispecies-model-of-cell-lineages" id="toc-sec-multispecies-model-of-cell-lineages" class="nav-link" data-scroll-target="#sec-multispecies-model-of-cell-lineages"><span class="header-section-number">2.4.5</span> A multispecies PDE model of cell lineages</a></li>
  </ul></li>
  <li><a href="#sec-models-of-differentiation-therapy" id="toc-sec-models-of-differentiation-therapy" class="nav-link" data-scroll-target="#sec-models-of-differentiation-therapy"><span class="header-section-number">2.5</span> Models of differentiation therapy</a>
  <ul class="collapse">
  <li><a href="#sec-differentiation-promoter-and-self-renewal-promoter" id="toc-sec-differentiation-promoter-and-self-renewal-promoter" class="nav-link" data-scroll-target="#sec-differentiation-promoter-and-self-renewal-promoter"><span class="header-section-number">2.5.1</span> Differentiation promoter and self-renewal promoter</a></li>
  <li><a href="#no-self-renewal-promoter" id="toc-no-self-renewal-promoter" class="nav-link" data-scroll-target="#no-self-renewal-promoter"><span class="header-section-number">2.5.2</span> No self-renewal promoter</a></li>
  <li><a href="#bmp4-in-glioma" id="toc-bmp4-in-glioma" class="nav-link" data-scroll-target="#bmp4-in-glioma"><span class="header-section-number">2.5.3</span> BMP4 in glioma</a></li>
  <li><a href="#summary-of-differentiation-therapy-results" id="toc-summary-of-differentiation-therapy-results" class="nav-link" data-scroll-target="#summary-of-differentiation-therapy-results"><span class="header-section-number">2.5.4</span> Summary of differentiation therapy results</a></li>
  </ul></li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion"><span class="header-section-number">2.6</span> Conclusion</a></li>
  </ul>
<div class="toc-actions"><ul><li><a href="https://github.com/Harbour-N/Second-year-report/edit/main/Lit_review.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/Harbour-N/Second-year-report/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span id="sec-lit-review" class="quarto-section-identifier"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Literature review</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p>In this chapter we present a brief literature review particularly focused of stem cells in cancer and the different modelling approaches that have been used.</p>
<section id="sec-introduction-lit" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="sec-introduction-lit"><span class="header-section-number">2.1</span> Introduction</h2>
<p>Stem cells are defined as cells that have the ability to perpetuate themselves through self-renewal and to generate mature cells of a particular tissue through differentiation <span class="citation" data-cites="reya2001">(<a href="references.html#ref-reya2001" role="doc-biblioref">Reya et al. 2001</a>)</span>. Stem cells are fundamental to tissue maintenance and repair; they also play a critical role in cancer development and in determining the outcomes of cancer treatment <span class="citation" data-cites="weiss2017a">(<a href="references.html#ref-weiss2017a" role="doc-biblioref">Weiss, Komarova, and Rodriguez-Brenes 2017</a>)</span>.</p>
</section>
<section id="sec-stem-cells-in-cancer" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="sec-stem-cells-in-cancer"><span class="header-section-number">2.2</span> Stem cells in cancer</h2>
<p>Perhaps the most important and useful property of stem cells is that of self-renewal. Self-renewal is crucial to stem cell function, because it is required by the majority of stem cells to persist for the lifetime of the animal. Moreover, whereas stem cells from different organs may vary in their developmental potential, all stem cells must self-renew and regulate the relative balance between self-renewal and differentiation. Understanding the regulation of normal stem cell self-renewal is also fundamental to understanding the regulation of cancer cell proliferation, because cancer can be considered to be a disease of unregulated self-renewal <span class="citation" data-cites="reya2001">(<a href="references.html#ref-reya2001" role="doc-biblioref">Reya et al. 2001</a>)</span>. Another distinguishing hallmark of stem cells is the ability to undergo asymmetric division, during which stem cells give rise to daughter cells of different fates, proliferative potential, size or other characteristics <span class="citation" data-cites="majumdar2020 hitomi2021">(<a href="references.html#ref-majumdar2020" role="doc-biblioref">Majumdar and Liu 2020</a>; <a href="references.html#ref-hitomi2021" role="doc-biblioref">Hitomi et al. 2021a</a>)</span>. Cancer stem cells (CSCs) generate such diverse progeny by executing multiple modes of cell division. Lineage-tracing experiments in glioma stem cells (GSCs) revealed that CSC undergo three main types of cell division: 1) Symmetric CSC self-renewing division, where a CSC produces two daughter CSCs; 2) symmetric differentiating division, where a CSC gives rise to two non-CSC daughter cells; 3) asymmetric division, where a CSC produces one CSC and one non-CSC. Additionally less than 1% of cell divisions resulted in cell death <span class="citation" data-cites="lathia2011">(<a href="references.html#ref-lathia2011" role="doc-biblioref">Lathia et al. 2011</a>)</span>. The types of CSC cell division are summarized in <a href="#fig-types-csc-div" class="quarto-xref">Figure&nbsp;<span>2.1</span></a>.</p>
<div id="fig-types-csc-div" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-types-csc-div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="images/lit_review/types_csc_div.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-types-csc-div-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2.1: The three main types of CSC cell division. Symmetric self-renewal results in two daughter CSCs. Symmetric differentiation results in two non-CSC daughter cells. Asymmetric differentiation results in one CSC and one non-CSC. Non-CSCs are denoted CC for cancer cells (created with BioRender.com).
</figcaption>
</figure>
</div>
<p>Numerous arguments suggest a stem-cell origin for human cancers. First, it is worth noting that stem cells possess many of the features that constitute the tumour phenotype, including self-renewal and essentially unlimited replicative potential <span class="citation" data-cites="hanahan2000">(<a href="references.html#ref-hanahan2000" role="doc-biblioref">Hanahan and Weinberg 2000</a>)</span>. Second, the mutations that initiate tumour formation seem to accumulate in cells that persist throughout a person’s life, as suggested by the exponential increase of cancer incidence with age <span class="citation" data-cites="meza2008">(<a href="references.html#ref-meza2008" role="doc-biblioref">Meza et al. 2008</a>)</span>. This is thought to reflect a requirement for between four and seven mutations in a single cell (and its progeny) to effect malignant transformation <span class="citation" data-cites="hanahan2000">(<a href="references.html#ref-hanahan2000" role="doc-biblioref">Hanahan and Weinberg 2000</a>)</span>. Similarly, cancer formation from cells that persist throughout life is suggested by an increased incidence in adults of skin tumours such as melanoma after higher childhood exposure to a mutagenic agent such as ultraviolet solar radiation <span class="citation" data-cites="balk2011">(<a href="references.html#ref-balk2011" role="doc-biblioref">Balk 2011</a>)</span>. Normal somatic stem cells are strong candidates for such persistent cells, an alternative explanation would be that a more mature cell undergoes a dedifferentiation event, reverting to a more primitive stem cell phenotype <span class="citation" data-cites="sell1993 reya2001">(<a href="references.html#ref-sell1993" role="doc-biblioref">Sell 1993</a>; <a href="references.html#ref-reya2001" role="doc-biblioref">Reya et al. 2001</a>)</span>.</p>
<p>A stem cell origin for human cancers was first identified in leukaemia, perhaps due to the high fraction of stem cells in the haematopoietic system, when it was discovered that some, but not all, cancer cells were able to initiate tumours of the blood <span class="citation" data-cites="taipale2001 lapidot1994 bonnet1997">(<a href="references.html#ref-taipale2001" role="doc-biblioref">Taipale and Beachy 2001</a>; <a href="references.html#ref-lapidot1994" role="doc-biblioref">Lapidot et al. 1994</a>; <a href="references.html#ref-bonnet1997" role="doc-biblioref">Bonnet and Dick 1997</a>)</span>. More recently CSCs have been identified in many solid tumours including breast, colon and brain <span class="citation" data-cites="al-hajj2003 ricci-vitiani2007 ignatova2002 hemmati2003 singh2004 galli2004">(<a href="references.html#ref-al-hajj2003" role="doc-biblioref">Al-Hajj et al. 2003</a>; <a href="references.html#ref-ricci-vitiani2007" role="doc-biblioref">Ricci-Vitiani et al. 2007</a>; <a href="references.html#ref-ignatova2002" role="doc-biblioref">Ignatova et al. 2002</a>; <a href="references.html#ref-hemmati2003" role="doc-biblioref">Hemmati et al. 2003</a>; <a href="references.html#ref-singh2004" role="doc-biblioref">Singh et al. 2004</a>; <a href="references.html#ref-galli2004" role="doc-biblioref">Galli et al. 2004</a>)</span>. In GBM, cells expressing the CD133 cell surface protein marker (also found on neural stem cells) have been identified as having stem cell properties <em>in vitro</em> <span class="citation" data-cites="singh2003">(<a href="references.html#ref-singh2003" role="doc-biblioref">Singh et al. 2003</a>)</span>. Furthermore, when tested using a xenograft assay, it was found that injection of as few as 100 CD133+ cells produced a tumour that could be serially transplanted and was phenotypically similar to the patient’s original tumour, while injection of <span class="math inline">\(10^5\)</span> CD133- cells survived in the host but did not cause a tumour <span class="citation" data-cites="singh2004">(<a href="references.html#ref-singh2004" role="doc-biblioref">Singh et al. 2004</a>)</span>. This provides strong evidence that there is a small subpopulation of glioma stem cells that have the unique ability to initiate tumours, while the majority of cells cannot.</p>
</section>
<section id="sec-cancer-stem-cells-and-treatment-resistance" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="sec-cancer-stem-cells-and-treatment-resistance"><span class="header-section-number">2.3</span> Cancer stem cells and treatment resistance</h2>
<p>Radiation therapy is the most common form of treatment across all cancers, with around 50% of all cancer patients receiving radiotherapy at some point in their treatment <span class="citation" data-cites="baskar2012">(<a href="references.html#ref-baskar2012" role="doc-biblioref">Baskar et al. 2012</a>)</span>. However, in addition to being tumour-initiating, CSCs are highly resistant to both radio- and chemo-therapy through preferential activation of the DNA damage checkpoint response and an increase in DNA repair capacity <span class="citation" data-cites="bao2006 tang2021 rich2007 schonberg2014">(<a href="references.html#ref-bao2006" role="doc-biblioref">Bao et al. 2006</a>; <a href="references.html#ref-tang2021" role="doc-biblioref">Tang et al. 2021a</a>; <a href="references.html#ref-rich2007" role="doc-biblioref">Rich 2007</a>; <a href="references.html#ref-schonberg2014" role="doc-biblioref">Schonberg et al. 2014</a>)</span>. In glioma, experimental results have shown that both in culture and mouse models CD133-expressing stem cells survive radiation in larger proportions than the majority of tumour cells which lack CD133 expression <span class="citation" data-cites="bao2006 gao2013">(<a href="references.html#ref-bao2006" role="doc-biblioref">Bao et al. 2006</a>; <a href="references.html#ref-gao2013" role="doc-biblioref">Gao et al. 2013</a>)</span>; these results suggest that CSC confer radio-resistance in GBM and ultimately are the source of tumour recurrence after radiation.</p>
<p>In addition to being resistant to treatment CSCs also engage in a synergistic relationship with the surrounding tumour microenvironment to promote angiogenesis, proliferation, migration, tumour survival, and immune evasion <span class="citation" data-cites="ma2018 rich2007">(<a href="references.html#ref-ma2018" role="doc-biblioref">Ma et al. 2018</a>; <a href="references.html#ref-rich2007" role="doc-biblioref">Rich 2007</a>)</span>. Taken together this highlights the important role CSCs play in determining tumour response to therapy. There is a desperate need for targeted therapies that either directly kill CSCs or sensitize CSCs to standard cytotoxic therapies in order to improve treatment outcomes.</p>
</section>
<section id="sec-mathematical-models-of-cancer-stem-cell-dynamics" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="sec-mathematical-models-of-cancer-stem-cell-dynamics"><span class="header-section-number">2.4</span> Mathematical models of cancer stem cell dynamics</h2>
<p>Many different mathematical models have been developed to model stem cell dynamics. Understanding CSC kinetics and interaction with their non-stem counterparts is still limited; theoretical and mathematical models may help elucidate their role in cancer progression and treatment response. Here we focus on a subset of models used in the literature that cover a wide range of modelling techniques and have particularly inspired our modelling used in the preprint presented at the end of this report.</p>
<p>Many of the following models use slightly different terminology to refer to the non-stem cell population such as cancer cell, progenitor cells or tumour cells, for clarity we will refer to non-stem cells always as cancer cells (CCs) throughout this review.</p>
<section id="sec-agent-based-model" class="level3" data-number="2.4.1">
<h3 data-number="2.4.1" class="anchored" data-anchor-id="sec-agent-based-model"><span class="header-section-number">2.4.1</span> Agent-based models</h3>
<p>In <span class="citation" data-cites="enderling2009">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>)</span> and <span class="citation" data-cites="gao2013">(<a href="references.html#ref-gao2013" role="doc-biblioref">Gao et al. 2013</a>)</span> the authors develop an agent-based model (ABM) to study the dynamics of CSCs and CCs in a tumour. It is assumed that tumours are a heterogenous mix of CSCs and CCs. Cells are considered as individual entities with a cell cycle and limited proliferation capacity <span class="math inline">\(\rho \in [0,\rho_\text{max}]\)</span>. CSCs have unlimited self-renewal, hence <span class="math inline">\(\rho_\text{max} = \infty\)</span>. At each cell division CSCs can undergo symmetric self-renewing division with probability <span class="math inline">\(\delta\)</span> or asymmetric division with probability <span class="math inline">\(1-\delta\)</span>. The proliferation capacity <span class="math inline">\(\rho\)</span> is decremented at each CC division and inherited by both daughter cells.</p>
<p>Simulations of the ABM model revealed the following key results:</p>
<ul>
<li>Tumours developing solely from CCs will inevitably die out, due to their limited proliferation capacity. Hence, CSCs are necessary for malignant tumour growth. This is consistent with experimental results showing only CSCs can initiate tumours <span class="citation" data-cites="lapidot1994 singh2004">(<a href="references.html#ref-lapidot1994" role="doc-biblioref">Lapidot et al. 1994</a>; <a href="references.html#ref-singh2004" role="doc-biblioref">Singh et al. 2004</a>)</span>.</li>
<li>Tumours started from a single CC could still persist for a long time as long-term dormant lesions, but due to space-limited growth remain small – well below the potential maximum size of <span class="math inline">\(2^{\rho_\text{max}}\)</span>. Significant growth only occurs once a CSC is initiated. In fact, even with a single CSC, the tumour may remain small for an extended period due to space-limited growth. This is consistent with the observation that many tumours remain dormant for many years before they start to grow <span class="citation" data-cites="sweeney1995 neves-e-castro2006 folkman2004">(<a href="references.html#ref-sweeney1995" role="doc-biblioref">Sweeney 1995</a>; <a href="references.html#ref-neves-e-castro2006" role="doc-biblioref">Neves-E-Castro 2006</a>; <a href="references.html#ref-folkman2004" role="doc-biblioref">Folkman and Kalluri 2004</a>)</span>.</li>
<li>A high rate of spontaneous death of CCs actually enables room for sufficient stem cell divisions to enrich the stem cell pool and drive tumour growth. This leads to what they call the “tumour growth paradox”, where counterintuitively an increase in the death rate of CCs decreases the total tumour size in the short term, in the long run it leads to an increase in the total tumour size as the tumour contains more CSCs.</li>
</ul>
<p>Mathematically the tumour growth paradox is defined as follows.</p>
<div id="def-tumor-growth-paradox" class="theorem definition">
<p><span class="theorem-title"><strong>Definition 2.1 (Tumour growth paradox)</strong></span> Let <span class="math inline">\(N_\alpha (t)\)</span> denote a total tumour population with death rate <span class="math inline">\(\alpha\)</span> for CCs. The population exhibits a tumour growth paradox if there exist death rates <span class="math inline">\(\alpha_1 &lt; \alpha_2\)</span> and times <span class="math inline">\(t_1,t_2, T\)</span> and <span class="math inline">\(T_0\)</span> such that</p>
<p><span id="eq-tumour-growth-paradox"><span class="math display">\[
\begin{aligned}
N_{\alpha_1}(t_1) = N_{\alpha_2}(t_2)&amp; \quad \text{and} \quad N_{\alpha_1}(t_1 + T) &lt; N_{\alpha_2}(t_2 + T) \\
&amp;\text{for} \quad (0&lt;T&lt;T_0),
\end{aligned}
\tag{2.1}\]</span></span></p>
</div>
</section>
<section id="sec-integro-differential-model" class="level3" data-number="2.4.2">
<h3 data-number="2.4.2" class="anchored" data-anchor-id="sec-integro-differential-model"><span class="header-section-number">2.4.2</span> Integro-differential model</h3>
<p>Following on from the ABM developed in <span class="citation" data-cites="enderling2009 gao2013">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>; <a href="references.html#ref-gao2013" role="doc-biblioref">Gao et al. 2013</a>)</span>, in <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> the authors develop an integro-differential equation version of the model, based on the same assumptions as in <span class="citation" data-cites="enderling2009 gao2013">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>; <a href="references.html#ref-gao2013" role="doc-biblioref">Gao et al. 2013</a>)</span> outlined in <a href="#sec-agent-based-model" class="quarto-xref"><span>Section 2.4.1</span></a>. Let <span class="math inline">\(u(x,t)\)</span> denote the density, in cells per unit space, and let <span class="math inline">\(v(x,t)\)</span> denote the density of CCs. Hence, the total tumour density is denoted <span class="math inline">\(N(x,t) = u(x,t) + v(x,t)\)</span>. For this analysis cells are assumed to be very small compared to the size of the tissue domain <span class="math inline">\(\Omega\)</span> (which we take without loss of generality to have unit volume). It is also assumed that cells cannot pile on top of each other so there is a maximum density of one cell per unit space, this implies <span class="math inline">\(N(x,t) \leq 1\)</span>. Cells can only proliferate if there is space to place the daughter cells, otherwise reproduction is inhibited (cellular quiescence). To model the spatial search for space, they define a nonlinear integral term, consistent with the ABM <span class="citation" data-cites="enderling2009 gao2013">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>; <a href="references.html#ref-gao2013" role="doc-biblioref">Gao et al. 2013</a>)</span> they assume that all cells can migrate randomly, which is modelled by simple diffusion. These assumptions lead to the following system of equations to describe CSC and CC dynamics:</p>
<p><span id="eq-integro-differential-CSC-model"><span class="math display">\[
\begin{aligned}
    \underbrace{\frac{\partial u(x,t)}{\partial t}}_\text{Rate of change CSCs} &amp;= \underbrace{D_u \nabla^2 u}_\text{Diffusion of CSCs} + \underbrace{\delta \gamma \int_{\Omega} k(x,y,N(x,t))u(y,t) dy}_\text{Self-renewal of CSCs} \\
    \underbrace{\frac{\partial v(x,t)}{\partial t}}_\text{Rate of change CCs} &amp;= \underbrace{D_v \nabla^2 v}_\text{Diffusion of CCs} + \underbrace{(1-\delta) \gamma \int_{\Omega} k(x,y,N(x,t))u(y,t) dy}_\text{Differentiation of CSCs} + \\
    &amp; \underbrace{\rho \int_{\Omega} k(x,y,N(x,t))v(y,t) dy}_\text{Proliferation of CCs} - \underbrace{\alpha v}_\text{Apoptosis of CCs}.
\end{aligned}
\tag{2.2}\]</span></span></p>
<p>The spatial distribution kernel <span class="math inline">\(k(x,y,N)\)</span> describes the rate of progeny contribution to location x for a cell at location y per “cell cycle time” i.e., the defined period between divisions of a freely cycling cell. Since greater density at <span class="math inline">\(x\)</span> would be expected to hinder progeny occupation it is assumed that <span class="math inline">\(k\)</span> is monotonically decreasing in <span class="math inline">\(N\)</span>, with <span class="math inline">\(k(x,y,N(x,t))=0\)</span> at <span class="math inline">\(N=1\)</span>. The number of cell cycle times per unit time of CSCs and CCs are denoted by <span class="math inline">\(\gamma\)</span> and <span class="math inline">\(\rho\)</span>, respectively, and for simplicity it is assumed that <span class="math inline">\(\gamma = \rho = 1\)</span> throughout. The parameter <span class="math inline">\(\delta\)</span> with <span class="math inline">\(0 \leq \delta \leq 1\)</span>, as in the ABM <span class="citation" data-cites="enderling2009 gao2013">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>; <a href="references.html#ref-gao2013" role="doc-biblioref">Gao et al. 2013</a>)</span>, denotes the fraction of CSC divisions that are symmetric self-renewal, while <span class="math inline">\(1-\delta\)</span> is the fraction of asymmetric divisions. The parameter <span class="math inline">\(\alpha\)</span> denotes the spontaneous death rate of CCs. Background cell motility is modelled by the diffusion coefficients <span class="math inline">\(D_u\)</span> and <span class="math inline">\(D_v\)</span> for CSCs and CCs, respectively. The system is considered to hold in a smooth bounded domain <span class="math inline">\(\Omega\)</span>, with homogeneous Neumann or Dirichlet boundary conditions.</p>
<p>Homogeneous Neumann boundary conditions correspond to a boundary that is impenetrable by cells, this could for example represent a tissue surrounded by membranes, smooth muscle tissue, or bone, and are given by</p>
<p><span id="eq-homogeneous-neumann-boundary-conditions"><span class="math display">\[
\frac{\partial u}{\partial n} = 0, \quad \frac{\partial v}{\partial n} = 0 \quad \text{on} \quad \partial \Omega,
\tag{2.3}\]</span></span></p>
<p>where <span class="math inline">\(\partial / \partial n\)</span> is the normal derivative at the boundary. The redistribution kernel can only redistribute cells within this domain <span class="math inline">\(\Omega\)</span>, hence we impose</p>
<p><span id="eq-redistribution-kernel-neumann-BC"><span class="math display">\[
k(x,y,N) = 0 \quad \text{for} \quad x \notin \Omega.
\tag{2.4}\]</span></span></p>
<p>Homogeneous Dirichlet boundary conditions correspond to a boundary that cells can freely leave but not re-enter again, for example this could represent intravasation into adjacent blood vessels, and are given by</p>
<p><span id="eq-homogeneous-dirichlet-boundary-conditions"><span class="math display">\[
u = 0, \quad v = 0 \quad \text{on} \quad \partial \Omega.
\tag{2.5}\]</span></span></p>
<p>The redistribution kernel describes transport of cells out of the domain but does not allow entering from the outside in, hence</p>
<p><span id="eq-redistribution-kernel-dirichlet-BC"><span class="math display">\[
k(x,y,N) = 0 \quad \text{for} \quad y \notin \Omega.
\tag{2.6}\]</span></span></p>
<p>Based on these two boundary conditions we can model any combination of domains such as partially covered by membranes, partially permeable membranes and adjacent blood vessels.</p>
</section>
<section id="sec-ode-model-reduction" class="level3" data-number="2.4.3">
<h3 data-number="2.4.3" class="anchored" data-anchor-id="sec-ode-model-reduction"><span class="header-section-number">2.4.3</span> ODE model reduction</h3>
<p>In order to analyse this model analytically <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> reduce the system of integro-differential equations (<a href="#eq-integro-differential-CSC-model" class="quarto-xref">Equation&nbsp;<span>2.2</span></a>) to a system of ordinary differential equations in the following way.</p>
<p><strong>Reduction 1: Progeny placement depends only on the density at the destination</strong><br>
In this case <span class="math inline">\(k(x,y,N(x,t)) = k(N(x,t))\)</span>. Introducing mean densities which, given that the domain <span class="math inline">\(\Omega\)</span> has unit volume, can be written as</p>
<p><span id="eq-density-mean"><span class="math display">\[
\bar{u}(t) = \int_{\Omega} u(y,t) dy, \quad \bar{v}(t) = \int_{\Omega} v(y,t) dy, \quad \bar{N}(t) = \bar{u}(t) + \bar{v}(t).
\tag{2.7}\]</span></span></p>
<p>Then <a href="#eq-integro-differential-CSC-model" class="quarto-xref">Equation&nbsp;<span>2.2</span></a> becomes:</p>
<p><span id="eq-first-reduction-ODE-CSC-model"><span class="math display">\[
\begin{aligned}
u_t(x,t) &amp;= D_u \nabla^2 u(x,t) + \delta   k(N(x,t))\bar{u}(t), \\
v_t(x,t) &amp;= D_v \nabla^2 v(x,t) + (1-\delta)   k(N(x,t))\bar{u}(t) + k(N(x,t))\bar{v}(t) - \alpha v(x,t).
\end{aligned}
\tag{2.8}\]</span></span></p>
<p><strong>Reduction 2: Density is uniform across the domain</strong><br>
If tumour growth is assumed uniform across the domain then, <span class="math inline">\(k(N(x,t)) = k(\bar{N}(t))\)</span> and <span class="math inline">\(u(x,t)\)</span> and <span class="math inline">\(v(x,t)\)</span> can be replaced with their spatially averaged values (<span class="math inline">\(\bar{u}(t)\)</span> and <span class="math inline">\(\bar{v}(t)\)</span>) and diffusion is zero everywhere. Therefore, <a href="#eq-first-reduction-ODE-CSC-model" class="quarto-xref">Equation&nbsp;<span>2.8</span></a> becomes:</p>
<p><span id="eq-ODE-CSC-model"><span class="math display">\[
\begin{aligned}
    \frac{d \bar{u}}{dt} &amp;= \delta  k(\bar{N}(t)) \bar{u}, \\
    \frac{d \bar{v}}{dt} &amp;= (1-\delta)  k(\bar{N}(t)) \bar{u} + k(\bar{N}(t)) \bar{v} - \alpha \bar{v}(t),
\end{aligned}
\tag{2.9}\]</span></span></p>
<p>where the volume filling constraint <span class="math inline">\(k(\bar{N})\)</span> is taken to be</p>
<p><span id="eq-volume-filling-constraint"><span class="math display">\[
k(\bar{N}) = \text{max} \left\{0, 1 - \bar{N}^\sigma \right\}, \quad \text{for} \quad \sigma &gt; 1.
\tag{2.10}\]</span></span></p>
<p>An exponent of <span class="math inline">\(\sigma = 1\)</span> corresponds to a linearly decreasing rate of occupancy for newborn cells as the total density <span class="math inline">\(\bar{N}\)</span> increases. Since cells are nonrigid, deformable and able to squeeze into available spaces <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> argue that <span class="math inline">\(\sigma &gt; 1\)</span> is appropriate and take it to be <span class="math inline">\(\sigma = 4\)</span>, in all their simulations.</p>
<p>Without a CSC population <span class="math inline">\(\bar{u}(t)\)</span>, the density of CCs <span class="math inline">\(\bar{v}(t)\)</span> satisfies the equation</p>
<p><span id="eq-ODE-TC-only-model"><span class="math display">\[
\frac{d\bar{v}}{dt}  =K(\bar{v}(t))\bar{v}(t) - \alpha \bar{v}(t).
\tag{2.11}\]</span></span></p>
<p>Since <span class="math inline">\(K(\bar{v}(t))\)</span> is a decreasing function of <span class="math inline">\(\bar{v}(t)\)</span> the CC population will die out when <span class="math inline">\(\alpha &gt; k(0)\)</span>. Note that this does not specifically set a limited proliferation capacity for CCs, as was the case in <span class="citation" data-cites="enderling2009">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>)</span>, rather if <span class="math inline">\(\alpha &gt; k(0)\)</span> then <span class="math inline">\(\alpha &gt; k(N)\)</span> for all <span class="math inline">\(N\)</span> hence the CC population will never survive on its own.</p>
<p>This simpler ODE model allows for analysis of the steady states, from which it can be shown that the pure stem cell steady state <span class="math inline">\((u,v)= (1,0)\)</span> is a global attractor. Therefore, this model predicts that for long times the tumour will consist of only stem cells. Intermediate tumour composition and the time at which the steady state is achieved are dependent on cell death rate <span class="math inline">\(\alpha\)</span>. The convergence to <span class="math inline">\((1,0)\)</span> is somewhat surprising as typically the CSC compartment is considered small, comprising only 1-3% of the total tumour <span class="citation" data-cites="bao2006">(<a href="references.html#ref-bao2006" role="doc-biblioref">Bao et al. 2006</a>)</span>. This suggests that the model may miss key biological dynamics of the CSCs such as some stem cell death and feedback regulation of symmetric / asymmetric division. However, <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> argue that this does not interfere with their analysis, as they are interested in the intermediate time dynamics of tumour initiation and growth, rather than the long-term behaviour as <span class="math inline">\(t \rightarrow \infty\)</span>.</p>
</section>
<section id="sec-stochastic-model" class="level3" data-number="2.4.4">
<h3 data-number="2.4.4" class="anchored" data-anchor-id="sec-stochastic-model"><span class="header-section-number">2.4.4</span> Stochastic model of CSC dynamics</h3>
<p>In <span class="citation" data-cites="turner2009">(<a href="references.html#ref-turner2009" role="doc-biblioref">Turner et al. 2009</a>)</span>, the authors develop a stochastic model for the dynamics of CSCs and CCs, particularly for the case of brain cancer. This stochastic model is particularly appropriate for situations in which small numbers of cells are present such as <em>in vitro</em> or in the early stages of tumour formation. In these cases stochastic fluctuations may have significant effects and cannot be neglected. To study larger populations the authors then derive a deterministic ODE model, based on the stochastic master equation, that describes the average number of CSCs and CCs.</p>
<p>The model assumptions on CSCs and CCs are largely similar as those given previously in <a href="#sec-agent-based-model" class="quarto-xref"><span>Section 2.4.1</span></a>. However, one key difference is that CSCs are assumed not to be immortal so have some probability of death. Defining <span class="math inline">\(p(n_s, n_p,t)\)</span> as the probability that there are exactly <span class="math inline">\(n_s\)</span> CSCs and <span class="math inline">\(n_p\)</span> CCs at time <span class="math inline">\(t\)</span>, the stochastic master equation is given by</p>
<p><span id="eq-stochastic-master-equation"><span class="math display">\[
\begin{aligned}
\frac{dp(n_s,n_p,t)}{dt} = &amp; \ \rho_s \left[ \underbrace{r_1 (n_s - 1) p(n_s - 1, n_p, t)}_{\text{symmetric self renewal of CSCs}} \right. \\
&amp; + \underbrace{r_2 n_s p(n_s, n_p - 1, t)}_{\text{asymmetric division of CSCs}} \\
&amp; + \underbrace{r_3 (n_s + 1) p(n_s + 1, n_p - 2, t)}_{\text{symmetric differentiation of CSCs}} \\
&amp; \left. - \underbrace{n_s p(n_s, n_p, t)}_\text{Overall division} \right] \\
&amp; + \underbrace{\Gamma_s \left[ (n_s + 1) p(n_s + 1, n_p, t) - n_s p(n_s, n_p, t) \right]}_\text{Apoptosis of CSCs} \\
&amp; + \underbrace{\Gamma_p \left[ (n_p + 1) p(n_s, n_p + 1, t) - n_p p(n_s, n_p, t) \right]}_\text{Apoptosis of CCs}.
\end{aligned}
\tag{2.12}\]</span></span></p>
<p>This model explicitly accounts for all three types of CSC division shown in <a href="#fig-types-csc-div" class="quarto-xref">Figure&nbsp;<span>2.1</span></a> where <span class="math inline">\(r_1=\)</span> symmetric self renewal probability, <span class="math inline">\(r_2=\)</span> asymmetric division probability, <span class="math inline">\(r_3=\)</span> symmetric differentiation probability, and <span class="math inline">\(\rho_s\)</span> represents the overall CSC division rate. While the model in <a href="#sec-integro-differential-model" class="quarto-xref"><span>Section 2.4.2</span></a> only accounts for asymmetric division it can be shown that these two formulations are equivalent for a large set of parameter values <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span>. The parameters <span class="math inline">\(\Gamma_s\)</span> and <span class="math inline">\(\Gamma_p\)</span> represent the rate of apoptosis for CSCs and CCs respectively. Due to the model’s stochastic nature, and the inclusion of a death probability for CSCs, it can be shown that the occurrence of a single CSC will not necessarily result in a tumour, even if the probability of self-renewal is greater than the probability of differentiation. This is in contrast to the previous models discussed <span class="citation" data-cites="enderling2009 hillen2013">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>; <a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> and to a deterministic version of this model (<a href="#eq-deterministic-master-equation" class="quarto-xref">Equation&nbsp;<span>2.13</span></a>) that would predict exponential growth of the tumour from a single CSC.</p>
<p>For larger cellular populations it becomes more challenging to simulate such a stochastic model and it becomes pertinent to consider the equations for the average number of CSCs and CCs. Defining the mean cellular populations <span class="math inline">\(S = &lt;n_s&gt;\)</span>, <span class="math inline">\(P = &lt;n_p&gt;\)</span> and <span class="math inline">\(r = r_1 - r_3\)</span> (i.e., the difference in symmetric self-renewal division and asymmetric differentiation division), the deterministic model is given by</p>
<p><span id="eq-deterministic-master-equation"><span class="math display">\[
\begin{aligned}
\frac{dS}{dt} &amp;= \rho_s r S - \Gamma_s S, \\
\frac{dP}{dt} &amp;= \rho_s (1-r) S - \Gamma_p P.
\end{aligned}
\tag{2.13}\]</span></span></p>
<p>This model is largely similar to the ODE model presented in <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span>, <a href="#eq-ODE-CSC-model" class="quarto-xref">Equation&nbsp;<span>2.9</span></a>. The main differences are that <a href="#eq-deterministic-master-equation" class="quarto-xref">Equation&nbsp;<span>2.13</span></a> includes a rate of CSC apoptosis <span class="math inline">\(\Gamma_s\)</span> and <a href="#eq-deterministic-master-equation" class="quarto-xref">Equation&nbsp;<span>2.13</span></a> does not account for completion for space. Although <span class="citation" data-cites="turner2009">(<a href="references.html#ref-turner2009" role="doc-biblioref">Turner et al. 2009</a>)</span> introduce this into a later version of the model, where they take <span class="math inline">\(\rho_s\)</span> to be given by <span class="math inline">\(\rho_s(S,P) = \rho_s(1 - c_s S - c_p P)\)</span>, where <span class="math inline">\(1/c_s\)</span> and <span class="math inline">\(1/c_p\)</span> are the limiting populations of stem and CCs respectively. However, a limitation of this approach is that it does not account for competition between CSCs and CCs for space and resources.</p>
</section>
<section id="sec-multispecies-model-of-cell-lineages" class="level3" data-number="2.4.5">
<h3 data-number="2.4.5" class="anchored" data-anchor-id="sec-multispecies-model-of-cell-lineages"><span class="header-section-number">2.4.5</span> A multispecies PDE model of cell lineages</h3>
<p>In <span class="citation" data-cites="youssefpour2012">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>)</span> the authors develop a multispecies PDE model for CSC lineage dynamics. This is the first model we have looked at that considers more than the two cell types, CSCs and CCs. Here the model is more complex and accounts for CSCs, committed progenitor cells, terminal cells, and dead cells. As with the previous models we have looked at it is assumes that differentiation and feedback processes link the cells’ lineage through self-renewal fractions and mitosis rates. The dependent variables in the model are the local volume fractions of the cell species denoted <span class="math inline">\(\phi_\text{CSC}, \phi_\text{CP}, \phi_\text{TC}, \phi_\text{DC}\)</span>, as well as healthy cells and water <span class="math inline">\(\phi_\text{H}, \phi_\text{W}\)</span>. Assuming there are no voids the sum of the volume fractions equals 1 and each cell type follows a conservation equation of the form</p>
<p><span id="eq-youssefpour-multispecies"><span class="math display">\[
\frac{\partial \phi}{\partial t} = - \nabla \cdot J - \nabla \cdot (u_s \phi) + S
\tag{2.14}\]</span></span></p>
<p>where <span class="math inline">\(\phi\)</span> denotes the volume fraction of the cell type, <span class="math inline">\(J\)</span> is generalized diffusion, <span class="math inline">\(u_s\)</span> is the mass-averaged velocity of the solid components, <span class="math inline">\(S\)</span> denotes the mass-exchange terms. This type of multiphase volume fraction approach has been widely used in cancer modelling <span class="citation" data-cites="lowengrub2010 breward2003 hubbard2013">(<a href="references.html#ref-lowengrub2010" role="doc-biblioref">Lowengrub et al. 2010</a>; <a href="references.html#ref-breward2003" role="doc-biblioref">Breward 2003</a>; <a href="references.html#ref-hubbard2013" role="doc-biblioref">Hubbard and Byrne 2013</a>)</span>. Simulation of this model show that the distributions of cell populations obtained from ABM model <span class="citation" data-cites="enderling2009">(<a href="references.html#ref-enderling2009" role="doc-biblioref">Enderling et al. 2009</a>)</span> and if this mutlspecies model are different. In the ABM CSCs tend to be located either at the center of small tumour clusters or distributed relatively uniformly throughout the tumour, some CSCs may migrate past the tumour boundary and generate new tumour clusters that eventually join the primary cluster crating an irregular boundary. On the other hand, the continuum model finds that CSCs tend to be located at the boundary of sufficient large tumour spheroids, this is consistent with some experiments <span class="citation" data-cites="vlashi2009">(<a href="references.html#ref-vlashi2009" role="doc-biblioref">Vlashi et al. 2009</a>)</span>. For smaller spheroids the CSCs may be more uniformly distributed. The patterns of CSCs and CCs are due to the cell signaling model implemented, detailed in <a href="#sec-differentiation-promoter-and-self-renewal-promoter" class="quarto-xref"><span>Section 2.5.1</span></a>, where the CSC self-renewal fraction is regulated by feedback factors produced by the different cells types <span class="citation" data-cites="youssefpour2012">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>)</span>.</p>
</section>
</section>
<section id="sec-models-of-differentiation-therapy" class="level2" data-number="2.5">
<h2 data-number="2.5" class="anchored" data-anchor-id="sec-models-of-differentiation-therapy"><span class="header-section-number">2.5</span> Models of differentiation therapy</h2>
<p>If, as with normal tissues, cellular phenotypic heterogeneity within tumours can be explained by a hierarchy of differentiation, with only a subset of stem-like cells capable of long-term self-renewal, this raises the prospect that signals promoting differentiation could be effective at driving malignant cells to a less aggressive and ideally post-mitotic differentiated state <span class="citation" data-cites="carén2016">(<a href="references.html#ref-carén2016" role="doc-biblioref">Carén, Beck, and Pollard 2016</a>)</span>. This differentiation therapy approach has seen success in acute promyelocytic leukemia (APL) where all-trans-retinoic acid (ATRA) can promote differentiation of CSCs and lead to complete remission <span class="citation" data-cites="yan2016 dethé2018">(<a href="references.html#ref-yan2016" role="doc-biblioref">Yan and Liu 2016</a>; <a href="references.html#ref-dethé2018" role="doc-biblioref">De Thé 2018</a>)</span>. In GBM, bone morphogenetic protein 4 (BMP4), a member of the TGF-<span class="math inline">\(\beta\)</span> superfamily, has shown potential as a differentiation therapeutic agent. BMP4 has been shown to drive differentiation of GSCs towards a predominantly glial (astrocytic) fate, to reduce GBM tumour burden <em>in vivo</em> and to improve survival in a mouse model of GBM <span class="citation" data-cites="nayak2020 piccirillo2006">(<a href="references.html#ref-nayak2020" role="doc-biblioref">Nayak et al. 2020</a>; <a href="references.html#ref-piccirillo2006" role="doc-biblioref">Piccirillo et al. 2006</a>)</span>. Despite its potential as a treatment option relatively few mathematical models have considered its possible effects on tumour growth <span class="citation" data-cites="youssefpour2012 bachman2013 turner2009">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>; <a href="references.html#ref-bachman2013" role="doc-biblioref">Bachman and Hillen 2013</a>; <a href="references.html#ref-turner2009" role="doc-biblioref">Turner et al. 2009</a>)</span>.</p>
<section id="sec-differentiation-promoter-and-self-renewal-promoter" class="level3" data-number="2.5.1">
<h3 data-number="2.5.1" class="anchored" data-anchor-id="sec-differentiation-promoter-and-self-renewal-promoter"><span class="header-section-number">2.5.1</span> Differentiation promoter and self-renewal promoter</h3>
<p>In <span class="citation" data-cites="youssefpour2012">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>)</span> they follow <span class="citation" data-cites="lander2009">(<a href="references.html#ref-lander2009" role="doc-biblioref">Lander et al. 2009</a>)</span> and assume that the proliferation and differentiation of CSCs are regulated by factors in the tumour microenvironemnt that feedback on self-renewal fractions and mitosis rates. In particular, they denote the differentiation promoter <span class="math inline">\(T\)</span> (for TGF-beta superfamily members) that reduces the probability of self-renewal for CSCs. They also account for self-renewal promoter <span class="math inline">\(W\)</span> which increases the probability of self-renewal of CSCs, as well as an inhibitor of <span class="math inline">\(W\)</span> denoted <span class="math inline">\(WI\)</span>. Possible self-renewal promoters include WNTs, Notch and Shh <span class="citation" data-cites="pannuti2010 bailey2007">(<a href="references.html#ref-pannuti2010" role="doc-biblioref">Pannuti et al. 2010</a>; <a href="references.html#ref-bailey2007" role="doc-biblioref">Bailey, Singh, and Hollingsworth 2007</a>)</span>. Using the notation introduced in <a href="#sec-multispecies-model-of-cell-lineages" class="quarto-xref"><span>Section 2.4.5</span></a>, they define the CSC self-renewal fraction as</p>
<p><span id="eq-youssefpour-Ps"><span class="math display">\[
P_s = P_\text{min} + (P_\text{max} - P_\text{min}) \left( \frac{\xi C_w}{1 + \xi C_W} \right)\left( \frac{1}{1 + \psi C_T} \right),
\tag{2.15}\]</span></span></p>
<p>where <span class="math inline">\(P_\text{min}\)</span> and <span class="math inline">\(P_\text{max}\)</span> are the minimum and maximum probability of CSC self-renewal, taken to be <span class="math inline">\(0.2\)</span> and <span class="math inline">\(1\)</span> respectively. <span class="math inline">\(C_W\)</span> and <span class="math inline">\(C_T\)</span> represent the concentrations of the self-renewal promoter and differentiation promoter respectively. The parameters <span class="math inline">\(\xi\)</span> and <span class="math inline">\(\psi\)</span> quantify the sensitivity of CSCs to the regulating proteins.</p>
<p>The concentrations of differentiation promoter and self-renewal promoter are then modelled as follows. It is assumed that <span class="math inline">\(T\)</span> is more diffuse than either <span class="math inline">\(W\)</span> or <span class="math inline">\(WI\)</span>. Therefore, on the time scale of cellular proliferation they make the quasi-steady-state assumption that time derivatives and advection of <span class="math inline">\(T\)</span> can be neglected. Thus, the quasi-steady reaction-diffusion equation for <span class="math inline">\(C_T\)</span> is given by</p>
<p><span id="eq-youssefpour-T"><span class="math display">\[
0 = \nabla \cdot (D_T \nabla C_T) - \left( \nu_{UT} \phi + \nu_{DT} \right) C_T + \nu_{PT}\phi_{TC}
\tag{2.16}\]</span></span></p>
<p>where <span class="math inline">\(D_T\)</span> is the diffusion coefficient, <span class="math inline">\(\nu_{UT}\)</span>, <span class="math inline">\(\nu_{DT}\)</span> and <span class="math inline">\(\nu_\text{PT}\)</span> are the uptake rate by CSCs, the rate of decay and the rate of production by the terminal cells, respectively.</p>
<p>To model the self-renewal promoter <span class="math inline">\(W\)</span> and its inhibitor <span class="math inline">\(WI\)</span> a generalized Gierer-Meinhard-Turing system of reaction advection diffusion equations is used, given by</p>
<p><span id="eq-youssefpour-W-WI"><span class="math display">\[
\begin{aligned}
&amp;\frac{\partial C_W}{\partial t} + \nabla \cdot (u_s C_W) = \nabla \cdot (D_W \nabla C_W) + f(C_W, C_{WI}), \\
&amp;\frac{\partial C_{WI}}{\partial t} + \nabla \cdot (u_s C_{WI}) = \nabla \cdot (D_{WI} \nabla C_{WI}) + g(C_W, C_{WI}),
\end{aligned}
\tag{2.17}\]</span></span></p>
<p>where</p>
<p><span id="eq-youssefpour-f-g"><span class="math display">\[
\begin{aligned}
f(C_W, C_{WI}) &amp;= \nu_{PW} \frac{C^2_W}{C_{WI}}C_0 \phi_\text{CSC} - \nu_{DW} C_W + u_0C_0 (\phi_\text{CSC} + \phi_\text{CP} + \phi_\text{TC}), \\
g(C_W, C_{WI}) &amp;= \nu_\text{PWI}C^2_W C_0 \phi_\text{CSC} - \nu_\text{DWI} C_{WI}.
\end{aligned}
\tag{2.18}\]</span></span></p>
<p>The parameters <span class="math inline">\(D_W\)</span> and <span class="math inline">\(D_{WI}\)</span> are the diffusion coefficients, <span class="math inline">\(\nu_\text{PW}\)</span>, <span class="math inline">\(\nu_\text{DW}\)</span> and <span class="math inline">\(\nu_\text{PWI}\)</span>, <span class="math inline">\(\nu_\text{DWI}\)</span> are the production and decay rates of <span class="math inline">\(W\)</span> and <span class="math inline">\(WI\)</span> respectively. The parameter <span class="math inline">\(u_0\)</span> represents a low-level source of <span class="math inline">\(W\)</span> from all the tumour cells.</p>
<p>To model differentiation therapy <span class="citation" data-cites="youssefpour2012">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>)</span> fix <span class="math inline">\(\psi = 0.5\)</span> and introduce an external source of <span class="math inline">\(T\)</span> i.e., a constant source term is added to <a href="#eq-youssefpour-T" class="quarto-xref">Equation&nbsp;<span>2.16</span></a>.</p>
</section>
<section id="no-self-renewal-promoter" class="level3" data-number="2.5.2">
<h3 data-number="2.5.2" class="anchored" data-anchor-id="no-self-renewal-promoter"><span class="header-section-number">2.5.2</span> No self-renewal promoter</h3>
<p>In <span class="citation" data-cites="bachman2013">(<a href="references.html#ref-bachman2013" role="doc-biblioref">Bachman and Hillen 2013</a>)</span> they follow <span class="citation" data-cites="youssefpour2012">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>)</span> and model differentiation therapy through a relationship between the average level of differentiation promoter, which they denote <span class="math inline">\(C_F\)</span> and the probability of CSC self-renewal <span class="math inline">\(P_s\)</span>. However, they do not include the effects of a CSC self-renewal promoting factor, thus</p>
<p><span id="eq-bachman-Ps"><span class="math display">\[
P_s(t) = P_\text{min} + (P_\text{max} - P_\text{min}) \left( \frac{1}{1 + \psi C_F(t)} \right),
\tag{2.19}\]</span></span></p>
<p>where the notation is the same as in <a href="#eq-youssefpour-Ps" class="quarto-xref">Equation&nbsp;<span>2.15</span></a>. Since <span class="citation" data-cites="bachman2013">(<a href="references.html#ref-bachman2013" role="doc-biblioref">Bachman and Hillen 2013</a>)</span> do not model endogenous production of differentiation promoter, <span class="math inline">\(C_F\)</span> solely represents the level of differentiation promoter prescribed during differentiation therapy. To address this lack of endogenous differentiation promoters they take <span class="math inline">\(P_\text{max} = 0.505\)</span> (which is equivalent to setting <span class="math inline">\(\delta = 0.01\)</span> as was done in <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span>) and <span class="math inline">\(P_\text{min} = 0.2\)</span>, as is done in <span class="citation" data-cites="youssefpour2012">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>)</span>.</p>
<p>As <span class="citation" data-cites="bachman2013">(<a href="references.html#ref-bachman2013" role="doc-biblioref">Bachman and Hillen 2013</a>)</span> use the ODE model developed in <span class="citation" data-cites="hillen2013">(<a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> they must also develop a submodel for the average level of differentiation promoter. As it is an ODE model they consider the average level of differentiation promoter within a spatially homogeous tumour <span class="math inline">\(C_F(t)\)</span>. It is assumed that the tumour resides within a spherical region of tissue and that differentiation promoter enters this area through the boundary. The differentiation promoter enters the region from the boundary and will diffuse very quickly and attain a steady state distribution over this region. To compute the value of <span class="math inline">\(C_F(t)\)</span> they solve the problem of diffusion over a sphere of radius <span class="math inline">\(R\)</span> and average the solution over the volume of the sphere. We use the lower case letter to describe the radial symmetric solution <span class="math inline">\(c_F(r,t)\)</span> of the following boundary value problem</p>
<p><span id="eq-bachman-CF"><span class="math display">\[
\begin{aligned}
\frac{\partial c_F}{\partial t} &amp;= \omega \left( \frac{\partial}{\partial r} \left(\frac{\partial c_F}{\partial r} \right) + \frac{2}{r} \frac{\partial c_f}{\partial r}   \right) \\
c_F(R,t) &amp;= C_{F0}(t),
\end{aligned}
\tag{2.20}\]</span></span></p>
<p>where <span class="math inline">\(\omega\)</span> is the effective diffusivity of the differentiation promoter. Before differentiation therapy begins <span class="math inline">\(C_{F0}(t) = 0\)</span>, when differentiation therapy begins the boundary condition on the sphere is set to <span class="math inline">\(C_{F0}(t) = 1\)</span>, and the promoter diffuses into the sphere. When differentiation therapy ends, the boundary condition is simply set to 0 and the promoter diffuses out of the sphere. They then set</p>
<p><span id="eq-bachman-CF2"><span class="math display">\[
C_F(t) = \frac{3}{R^3} = \int^R_0 c_F (r,t) r^2 dr.
\tag{2.21}\]</span></span></p>
</section>
<section id="bmp4-in-glioma" class="level3" data-number="2.5.3">
<h3 data-number="2.5.3" class="anchored" data-anchor-id="bmp4-in-glioma"><span class="header-section-number">2.5.3</span> BMP4 in glioma</h3>
<p>In the previous models <span class="citation" data-cites="youssefpour2012 hillen2013">(<a href="references.html#ref-youssefpour2012" role="doc-biblioref">Youssefpour et al. 2012</a>; <a href="references.html#ref-hillen2013" role="doc-biblioref">Hillen, Enderling, and Hahnfeldt 2013</a>)</span> they considered a general differentiation promoter, in <span class="citation" data-cites="turner2009">(<a href="references.html#ref-turner2009" role="doc-biblioref">Turner et al. 2009</a>)</span> they consider the specific differentiation promoter BMP4 in GBM. As in the case for the general differentiation promoter they interpret the effects of BMP4 as decreasing the net symmetric division rate <span class="math inline">\(r\)</span> (following the notation used in <a href="#sec-stochastic-model" class="quarto-xref"><span>Section 2.4.4</span></a>). Based on <span class="citation" data-cites="piccirillo2006">(<a href="references.html#ref-piccirillo2006" role="doc-biblioref">Piccirillo et al. 2006</a>)</span> they estimate that from a pre-treatment value of <span class="math inline">\(r = 0.1\)</span> the effect of BMP4 is to reduce <span class="math inline">\(r\)</span> to <span class="math inline">\(-0.1\)</span>, note that following the notation used in <a href="#sec-stochastic-model" class="quarto-xref"><span>Section 2.4.4</span></a> <span class="math inline">\(r\)</span> is defined as <span class="math inline">\(r = r_1-r_3\)</span> so a change of <span class="math inline">\(r\)</span> to negative represents a both an increase in the proportion of symmetric differentiation divisions and a decrease in symmetric self-renewal divisions. To model differentiation therapy the parameter value <span class="math inline">\(r\)</span> is simply switched between these two values for the duration of BMP4 exposure.</p>
</section>
<section id="summary-of-differentiation-therapy-results" class="level3" data-number="2.5.4">
<h3 data-number="2.5.4" class="anchored" data-anchor-id="summary-of-differentiation-therapy-results"><span class="header-section-number">2.5.4</span> Summary of differentiation therapy results</h3>
<p>All the models compare 3 different treatment cases radiation alone, differentiation therapy alone and combination therapy. Importantly, as has been shown, all models assume that CSCs are less sensitive to radiation than CCs <span class="citation" data-cites="bao2006 tang2021a hitomi2021a">(<a href="references.html#ref-bao2006" role="doc-biblioref">Bao et al. 2006</a>; <a href="references.html#ref-tang2021a" role="doc-biblioref">Tang et al. 2021b</a>; <a href="references.html#ref-hitomi2021a" role="doc-biblioref">Hitomi et al. 2021b</a>)</span>. Despite the slight differences in assumptions and implementation, all models find similar results. Radiotherapy alone fails as some CSCs survive and are able to repopulate the tumour. In fact, all models show an extension of the “tumour growth paradox” which we term the “tumour treatment paradox”. When treating with radiation alone the fraction of CSCs increases, since the CCs are more susceptible they significantly reduce their numbers leaving more room for CSCs. This CSC enriched post-treatment state allows much more rapid re-growth of the tumour. This suggests that current standard of care treatment selects for the more resistant CSCs. Thus treatment often facilitates more rapid and aggressive tumour recurrence. Differentiation therapy alone can successfully eradicate the tumour, however, given all models assume that CSCs can only transform into CCs through cellular division, rather than direct transition, large intermediate values of total tumour size may be reached using this approach. Combination of differentiation and radiotherapy out performed either single therapies, often showing that the tumour can be driven to much smaller sizes and potentially extinction. This is because the differentiation agent induces CSCs to turn into CCs which then can be killed by traditional radiation therapy. This combination therapy can be considered a new class of strategies for cancer therapy known as evolutionary steering approaches. Rather than reactively altering treatment as resistance is acquired we proactively select our treatment to minimise resistance and increase chance of extinction <span class="citation" data-cites="enriquez-navas2015 acar2020">(<a href="references.html#ref-enriquez-navas2015" role="doc-biblioref">Enriquez-Navas, Wojtkowiak, and Gatenby 2015</a>; <a href="references.html#ref-acar2020" role="doc-biblioref">Acar et al. 2020</a>)</span>.</p>
</section>
</section>
<section id="conclusion" class="level2" data-number="2.6">
<h2 data-number="2.6" class="anchored" data-anchor-id="conclusion"><span class="header-section-number">2.6</span> Conclusion</h2>
<p>In summary, mathematical models of CSC dynamics provide valuable insights into tumour growth, dormancy, and response to treatment. While the models reviewed here vary in their complexity and underlying assumptions, they all highlight the critical role of CSCs in tumour maintenance, treatment resistance and tumour recurrence. Mathematical modelling provides a unique tool for analysing the non-trivial interaction between CSCs and CCs that can arise from fairly simple assumptions, allowing us to verify or generate new hypothesis in order to better understand the role of CSCs.</p>
<p>The differentiation therapy approach aims to drive CSCs within tumours toward a differentiated state, potentially mitigating their self-renewal capacity and aggressiveness. Despite this, many challenges remain in developing differentiation therapies that can be used in a clinical setting <span class="citation" data-cites="carén2016">(<a href="references.html#ref-carén2016" role="doc-biblioref">Carén, Beck, and Pollard 2016</a>)</span>. One key challenge particularly in GBM is delivery of differentiation therapy to the tumour microenvironment, due to the blood brain barrier (BBB) which limits transport into the brain. New delivery mechanism to combat this are being developed such as adipose derived mesenchymal stem cells, containing BMP4 in nanoparticles. These can be delivered directly into the tumour at the time of resection or systemically as they have been shown to cross the BBB <span class="citation" data-cites="li2014a mangraviti2016a pendleton2013a">(<a href="references.html#ref-li2014a" role="doc-biblioref">Li et al. 2014</a>; <a href="references.html#ref-mangraviti2016a" role="doc-biblioref">Mangraviti et al. 2016</a>; <a href="references.html#ref-pendleton2013a" role="doc-biblioref">Pendleton et al. 2013</a>)</span>. Additionally as with all therapies there is vast heterogeneity between patient response to differentiation therapy, indicating the importance of identifying biomarkers for sensitivity to such therapies. In work in preparation it has been shown that GBM cell lines that did not express pRBS were unresponsive to differentiation therapy <span class="citation" data-cites="fariasa">(<a href="references.html#ref-fariasa" role="doc-biblioref">Farias et al., n.d.</a>)</span>. Mathematical modelling may also help improve the effectiveness of differentiation therapy strategies optimising its delivery in combination with standard cytotoxic therapies as well as elucidating the mechanism through which CSCs drive tumour growth.</p>
<p>In the next chapter I present our preprint, looking at a model for BMP4 induced differentiation therapy in GBM, which is available on bioR<span class="math inline">\(\chi\)</span>iv <a href="https://doi.org/10.1101/2024.08.22.609156">Virtual Clinical Trials of BMP4 Differentiation Therapy: Digital Twins to Aid Glioblastoma Trial Design</a>.</p>


<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list" style="display: none">
<div id="ref-acar2020" class="csl-entry" role="listitem">
Acar, Ahmet, Daniel Nichol, Javier Fernandez-Mateos, George D. Cresswell, Iros Barozzi, Sung Pil Hong, Nicholas Trahearn, et al. 2020. <span>“Exploiting Evolutionary Steering to Induce Collateral Drug Sensitivity in Cancer.”</span> <em>Nature Communications</em> 11 (1): 1923. <a href="https://doi.org/10.1038/s41467-020-15596-z">https://doi.org/10.1038/s41467-020-15596-z</a>.
</div>
<div id="ref-al-hajj2003" class="csl-entry" role="listitem">
Al-Hajj, Muhammad, Max S. Wicha, Adalberto Benito-Hernandez, Sean J. Morrison, and Michael F. Clarke. 2003. <span>“Prospective Identification of Tumorigenic Breast Cancer Cells.”</span> <em>Proceedings of the National Academy of Sciences</em> 100 (7): 3983–88. <a href="https://doi.org/10.1073/pnas.0530291100">https://doi.org/10.1073/pnas.0530291100</a>.
</div>
<div id="ref-bachman2013" class="csl-entry" role="listitem">
Bachman, Jeff W. N., and Thomas Hillen. 2013. <span>“Mathematical Optimization of the Combination of Radiation and Differentiation Therapies for Cancer.”</span> <em>Frontiers in Oncology</em> 3. <a href="https://doi.org/10.3389/fonc.2013.00052">https://doi.org/10.3389/fonc.2013.00052</a>.
</div>
<div id="ref-bailey2007" class="csl-entry" role="listitem">
Bailey, Jennifer M., Pankaj K. Singh, and Michael A. Hollingsworth. 2007. <span>“Cancer Metastasis Facilitated by Developmental Pathways: Sonic Hedgehog, Notch, and Bone Morphogenic Proteins.”</span> <em>Journal of Cellular Biochemistry</em> 102 (4): 829–39. <a href="https://doi.org/10.1002/jcb.21509">https://doi.org/10.1002/jcb.21509</a>.
</div>
<div id="ref-balk2011" class="csl-entry" role="listitem">
Balk, Sophie J. 2011. <span>“Ultraviolet Radiation: A Hazard to Children and Adolescents.”</span> <em>Pediatrics</em> 127 (3): e791–817. <a href="https://doi.org/10.1542/peds.2010-3502">https://doi.org/10.1542/peds.2010-3502</a>.
</div>
<div id="ref-bao2006" class="csl-entry" role="listitem">
Bao, Shideng, Qiulian Wu, Roger E. McLendon, Yueling Hao, Qing Shi, Anita B. Hjelmeland, Mark W. Dewhirst, Darell D. Bigner, and Jeremy N. Rich. 2006. <span>“Glioma Stem Cells Promote Radioresistance by Preferential Activation of the DNA Damage Response.”</span> <em>Nature</em> 444 (7120): 756–60. <a href="https://doi.org/10.1038/nature05236">https://doi.org/10.1038/nature05236</a>.
</div>
<div id="ref-baskar2012" class="csl-entry" role="listitem">
Baskar, Rajamanickam, Kuo Ann Lee, Richard Yeo, and Kheng-Wei Yeoh. 2012. <span>“Cancer and Radiation Therapy: Current Advances and Future Directions.”</span> <em>International Journal of Medical Sciences</em> 9 (3): 193–99. <a href="https://doi.org/10.7150/ijms.3635">https://doi.org/10.7150/ijms.3635</a>.
</div>
<div id="ref-bonnet1997" class="csl-entry" role="listitem">
Bonnet, Dominique, and John E. Dick. 1997. <span>“Human Acute Myeloid Leukemia Is Organized as a Hierarchy That Originates from a Primitive Hematopoietic Cell.”</span> <em>Nature Medicine</em> 3 (7): 730–37. <a href="https://doi.org/10.1038/nm0797-730">https://doi.org/10.1038/nm0797-730</a>.
</div>
<div id="ref-breward2003" class="csl-entry" role="listitem">
Breward, C. 2003. <span>“A Multiphase Model Describing Vascular Tumour Growth.”</span> <em>Bulletin of Mathematical Biology</em> 65 (4): 609–40. <a href="https://doi.org/10.1016/S0092-8240(03)00027-2">https://doi.org/10.1016/S0092-8240(03)00027-2</a>.
</div>
<div id="ref-carén2016" class="csl-entry" role="listitem">
Carén, Helena, Stephan Beck, and Steven M. Pollard. 2016. <span>“Differentiation Therapy for Glioblastoma <span></span> Too Many Obstacles?”</span> <em>Molecular &amp; Cellular Oncology</em> 3 (2): e1124174. <a href="https://doi.org/10.1080/23723556.2015.1124174">https://doi.org/10.1080/23723556.2015.1124174</a>.
</div>
<div id="ref-dethé2018" class="csl-entry" role="listitem">
De Thé, Hugues. 2018. <span>“Differentiation Therapy Revisited.”</span> <em>Nature Reviews Cancer</em> 18 (2): 117–27. <a href="https://doi.org/10.1038/nrc.2017.103">https://doi.org/10.1038/nrc.2017.103</a>.
</div>
<div id="ref-enderling2009" class="csl-entry" role="listitem">
Enderling, Heiko, Alexander R. A. Anderson, Mark A. J. Chaplain, Afshin Beheshti, Lynn Hlatky, and Philip Hahnfeldt. 2009. <span>“Paradoxical Dependencies of Tumor Dormancy and Progression on Basic Cell Kinetics.”</span> <em>Cancer Research</em> 69 (22): 8814–21. <a href="https://doi.org/10.1158/0008-5472.CAN-09-2115">https://doi.org/10.1158/0008-5472.CAN-09-2115</a>.
</div>
<div id="ref-enriquez-navas2015" class="csl-entry" role="listitem">
Enriquez-Navas, Pedro M., Jonathan W. Wojtkowiak, and Robert A. Gatenby. 2015. <span>“Application of Evolutionary Principles to Cancer Therapy.”</span> <em>Cancer Research</em> 75 (22): 4675–80. <a href="https://doi.org/10.1158/0008-5472.CAN-15-1337">https://doi.org/10.1158/0008-5472.CAN-15-1337</a>.
</div>
<div id="ref-fariasa" class="csl-entry" role="listitem">
Farias, Virgina, Vinitha Rani, Natanael Zacro, Mieu Brooks, Naveen Nagaiah, Henry Ruiz-Garcia, Rachel Sarabia-Estrada, et al. n.d. <span>“BMP4-Secreting MSCs Derived from Human Adipose Tissue Radiosensitize Glioblastoma Stem Cells via Downregulation of OLIG2 Signaling Pathway.”</span> <em>In Preparation</em>.
</div>
<div id="ref-folkman2004" class="csl-entry" role="listitem">
Folkman, Judah, and Raghu Kalluri. 2004. <span>“Cancer Without Disease.”</span> <em>Nature</em> 427 (6977): 787–87. <a href="https://doi.org/10.1038/427787a">https://doi.org/10.1038/427787a</a>.
</div>
<div id="ref-galli2004" class="csl-entry" role="listitem">
Galli, Rossella, Elena Binda, Ugo Orfanelli, Barbara Cipelletti, Angela Gritti, Simona De Vitis, Roberta Fiocco, Chiara Foroni, Francesco Dimeco, and Angelo Vescovi. 2004. <span>“Isolation and Characterization of Tumorigenic, Stem-Like Neural Precursors from Human Glioblastoma.”</span> <em>Cancer Research</em> 64 (19): 7011–21. <a href="https://doi.org/10.1158/0008-5472.CAN-04-1364">https://doi.org/10.1158/0008-5472.CAN-04-1364</a>.
</div>
<div id="ref-gao2013" class="csl-entry" role="listitem">
Gao, Xuefeng, J. Tyson McDonald, Lynn Hlatky, and Heiko Enderling. 2013. <span>“Acute and Fractionated Irradiation Differentially Modulate Glioma Stem Cell Division Kinetics.”</span> <em>Cancer Research</em> 73 (5): 1481–90. <a href="https://doi.org/10.1158/0008-5472.CAN-12-3429">https://doi.org/10.1158/0008-5472.CAN-12-3429</a>.
</div>
<div id="ref-hanahan2000" class="csl-entry" role="listitem">
Hanahan, Douglas, and Robert A Weinberg. 2000. <span>“The Hallmarks of Cancer.”</span> <em>Cell</em> 100 (1): 57–70. <a href="https://doi.org/10.1016/S0092-8674(00)81683-9">https://doi.org/10.1016/S0092-8674(00)81683-9</a>.
</div>
<div id="ref-hemmati2003" class="csl-entry" role="listitem">
Hemmati, Houman D., Ichiro Nakano, Jorge A. Lazareff, Michael Masterman-Smith, Daniel H. Geschwind, Marianne Bronner-Fraser, and Harley I. Kornblum. 2003. <span>“Cancerous Stem Cells Can Arise from Pediatric Brain Tumors.”</span> <em>Proceedings of the National Academy of Sciences</em> 100 (25): 15178–83. <a href="https://doi.org/10.1073/pnas.2036535100">https://doi.org/10.1073/pnas.2036535100</a>.
</div>
<div id="ref-hillen2013" class="csl-entry" role="listitem">
Hillen, Thomas, Heiko Enderling, and Philip Hahnfeldt. 2013. <span>“The Tumor Growth Paradox and Immune System-Mediated Selection for Cancer Stem Cells.”</span> <em>Bulletin of Mathematical Biology</em> 75 (1): 161–84. <a href="https://doi.org/10.1007/s11538-012-9798-x">https://doi.org/10.1007/s11538-012-9798-x</a>.
</div>
<div id="ref-hitomi2021" class="csl-entry" role="listitem">
Hitomi, Masahiro, Anastasia P. Chumakova, Daniel J. Silver, Arnon M. Knudsen, W. Dean Pontius, Stephanie Murphy, Neha Anand, Bjarne W. Kristensen, and Justin D. Lathia. 2021a. <span>“Asymmetric Cell Division Promotes Therapeutic Resistance in Glioblastoma Stem Cells.”</span> <em>JCI Insight</em> 6 (3): e130510. <a href="https://doi.org/10.1172/jci.insight.130510">https://doi.org/10.1172/jci.insight.130510</a>.
</div>
<div id="ref-hitomi2021a" class="csl-entry" role="listitem">
———. 2021b. <span>“Asymmetric Cell Division Promotes Therapeutic Resistance in Glioblastoma Stem Cells.”</span> <em>JCI Insight</em> 6 (3): e130510. <a href="https://doi.org/10.1172/jci.insight.130510">https://doi.org/10.1172/jci.insight.130510</a>.
</div>
<div id="ref-hubbard2013" class="csl-entry" role="listitem">
Hubbard, M. E., and H. M. Byrne. 2013. <span>“Multiphase Modelling of Vascular Tumour Growth in Two Spatial Dimensions.”</span> <em>Journal of Theoretical Biology</em> 316 (January): 70–89. <a href="https://doi.org/10.1016/j.jtbi.2012.09.031">https://doi.org/10.1016/j.jtbi.2012.09.031</a>.
</div>
<div id="ref-ignatova2002" class="csl-entry" role="listitem">
Ignatova, Tatyana N., Valery G. Kukekov, Eric D. Laywell, Oleg N. Suslov, Frank D. Vrionis, and Dennis A. Steindler. 2002. <span>“Human Cortical Glial Tumors Contain Neural Stem<span>-</span>Like Cells Expressing Astroglial and Neuronal Markers in Vitro.”</span> <em>Glia</em> 39 (3): 193–206. <a href="https://doi.org/10.1002/glia.10094">https://doi.org/10.1002/glia.10094</a>.
</div>
<div id="ref-lander2009" class="csl-entry" role="listitem">
Lander, Arthur D, Kimberly K Gokoffski, Frederic Y. M Wan, Qing Nie, and Anne L Calof. 2009. <span>“Cell Lineages and the Logic of Proliferative Control.”</span> Edited by Charles F Stevens. <em>PLoS Biology</em> 7 (1): e1000015. <a href="https://doi.org/10.1371/journal.pbio.1000015">https://doi.org/10.1371/journal.pbio.1000015</a>.
</div>
<div id="ref-lapidot1994" class="csl-entry" role="listitem">
Lapidot, Tsvee, Christian Sirard, Josef Vormoor, Barbara Murdoch, Trang Hoang, Julio Caceres-Cortes, Mark Minden, Bruce Paterson, Michael A. Caligiuri, and John E. Dick. 1994. <span>“A Cell Initiating Human Acute Myeloid Leukaemia After Transplantation into SCID Mice.”</span> <em>Nature</em> 367 (6464): 645–48. <a href="https://doi.org/10.1038/367645a0">https://doi.org/10.1038/367645a0</a>.
</div>
<div id="ref-lathia2011" class="csl-entry" role="listitem">
Lathia, J D, M Hitomi, J Gallagher, S P Gadani, J Adkins, A Vasanji, L Liu, et al. 2011. <span>“Distribution of CD133 Reveals Glioma Stem Cells Self-Renew Through Symmetric and Asymmetric Cell Divisions.”</span> <em>Cell Death &amp; Disease</em> 2 (9): e200–200. <a href="https://doi.org/10.1038/cddis.2011.80">https://doi.org/10.1038/cddis.2011.80</a>.
</div>
<div id="ref-li2014a" class="csl-entry" role="listitem">
Li, Qian, Olindi Wijesekera, Sussan J. Salas, Joanna Y. Wang, Mingxin Zhu, Colette Aprhys, Kaisorn L. Chaichana, et al. 2014. <span>“Mesenchymal Stem Cells from Human Fat Engineered to Secrete BMP4 Are Nononcogenic, Suppress Brain Cancer, and Prolong Survival.”</span> <em>Clinical Cancer Research</em> 20 (9): 2375–87. <a href="https://doi.org/10.1158/1078-0432.CCR-13-1415">https://doi.org/10.1158/1078-0432.CCR-13-1415</a>.
</div>
<div id="ref-lowengrub2010" class="csl-entry" role="listitem">
Lowengrub, J S, H B Frieboes, F Jin, Y-L Chuang, X Li, P Macklin, S M Wise, and V Cristini. 2010. <span>“Nonlinear Modelling of Cancer: Bridging the Gap Between Cells and Tumours.”</span> <em>Nonlinearity</em> 23 (1): R1–91. <a href="https://doi.org/10.1088/0951-7715/23/1/R01">https://doi.org/10.1088/0951-7715/23/1/R01</a>.
</div>
<div id="ref-ma2018" class="csl-entry" role="listitem">
Ma, Qianquan, Wenyong Long, Changsheng Xing, Junjun Chu, Mei Luo, Helen Y. Wang, Qing Liu, and Rong-Fu Wang. 2018. <span>“Cancer Stem Cells and Immunosuppressive Microenvironment in Glioma.”</span> <em>Frontiers in Immunology</em> 9 (December): 2924. <a href="https://doi.org/10.3389/fimmu.2018.02924">https://doi.org/10.3389/fimmu.2018.02924</a>.
</div>
<div id="ref-majumdar2020" class="csl-entry" role="listitem">
Majumdar, Sreemita, and Song-Tao Liu. 2020. <span>“Cell Division Symmetry Control and Cancer Stem Cells.”</span> <em>AIMS Molecular Science</em> 7 (2): 82–101. <a href="https://doi.org/10.3934/molsci.2020006">https://doi.org/10.3934/molsci.2020006</a>.
</div>
<div id="ref-mangraviti2016a" class="csl-entry" role="listitem">
Mangraviti, Antonella, Stephany Y. Tzeng, David Gullotti, Kristen L. Kozielski, Jennifer E. Kim, Michael Seng, Sara Abbadi, et al. 2016. <span>“Non-Virally Engineered Human Adipose Mesenchymal Stem Cells Produce BMP4, Target Brain Tumors, and Extend Survival.”</span> <em>Biomaterials</em> 100 (September): 53–66. <a href="https://doi.org/10.1016/j.biomaterials.2016.05.025">https://doi.org/10.1016/j.biomaterials.2016.05.025</a>.
</div>
<div id="ref-meza2008" class="csl-entry" role="listitem">
Meza, Rafael, Jihyoun Jeon, Suresh H. Moolgavkar, and E. Georg Luebeck. 2008. <span>“Age-Specific Incidence of Cancer: Phases, Transitions, and Biological Implications.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (42): 16284–89. <a href="https://doi.org/10.1073/pnas.0801151105">https://doi.org/10.1073/pnas.0801151105</a>.
</div>
<div id="ref-nayak2020" class="csl-entry" role="listitem">
Nayak, Sonali, Ashorne Mahenthiran, Yongyong Yang, Mark McClendon, Barbara Mania-Farnell, Charles David James, John A. Kessler, et al. 2020. <span>“Bone Morphogenetic Protein 4 Targeting Glioma Stem-Like Cells for Malignant Glioma Treatment: Latest Advances and Implications for Clinical Application.”</span> <em>Cancers</em> 12 (2): 516. <a href="https://doi.org/10.3390/cancers12020516">https://doi.org/10.3390/cancers12020516</a>.
</div>
<div id="ref-neves-e-castro2006" class="csl-entry" role="listitem">
Neves-E-Castro, Manuel. 2006. <span>“Why Do Some Breast Cancer Cells Remain Dormant?”</span> <em>Gynecological Endocrinology</em> 22 (4): 190–97. <a href="https://doi.org/10.1080/09513590600624374">https://doi.org/10.1080/09513590600624374</a>.
</div>
<div id="ref-pannuti2010" class="csl-entry" role="listitem">
Pannuti, Antonio, Kimberly Foreman, Paola Rizzo, Clodia Osipo, Todd Golde, Barbara Osborne, and Lucio Miele. 2010. <span>“Targeting Notch to Target Cancer Stem Cells.”</span> <em>Clinical Cancer Research</em> 16 (12): 3141–52. <a href="https://doi.org/10.1158/1078-0432.CCR-09-2823">https://doi.org/10.1158/1078-0432.CCR-09-2823</a>.
</div>
<div id="ref-pendleton2013a" class="csl-entry" role="listitem">
Pendleton, Courtney, Qian Li, David A. Chesler, Kristy Yuan, Hugo Guerrero-Cazares, and Alfredo Quinones-Hinojosa. 2013. <span>“Mesenchymal Stem Cells Derived from Adipose Tissue Vs Bone Marrow: In Vitro Comparison of Their Tropism Towards Gliomas.”</span> Edited by Joseph Najbauer. <em>PLoS ONE</em> 8 (3): e58198. <a href="https://doi.org/10.1371/journal.pone.0058198">https://doi.org/10.1371/journal.pone.0058198</a>.
</div>
<div id="ref-piccirillo2006" class="csl-entry" role="listitem">
Piccirillo, S. G. M., B. A. Reynolds, N. Zanetti, G. Lamorte, E. Binda, G. Broggi, H. Brem, A. Olivi, F. Dimeco, and A. L. Vescovi. 2006. <span>“Bone Morphogenetic Proteins Inhibit the Tumorigenic Potential of Human Brain Tumour-Initiating Cells.”</span> <em>Nature</em> 444 (7120): 761–65. <a href="https://doi.org/10.1038/nature05349">https://doi.org/10.1038/nature05349</a>.
</div>
<div id="ref-reya2001" class="csl-entry" role="listitem">
Reya, Tannishtha, Sean J. Morrison, Michael F. Clarke, and Irving L. Weissman. 2001. <span>“Stem Cells, Cancer, and Cancer Stem Cells.”</span> <em>Nature</em> 414 (6859): 105–11. <a href="https://doi.org/10.1038/35102167">https://doi.org/10.1038/35102167</a>.
</div>
<div id="ref-ricci-vitiani2007" class="csl-entry" role="listitem">
Ricci-Vitiani, Lucia, Dario G. Lombardi, Emanuela Pilozzi, Mauro Biffoni, Matilde Todaro, Cesare Peschle, and Ruggero De Maria. 2007. <span>“Identification and Expansion of Human Colon-Cancer-Initiating Cells.”</span> <em>Nature</em> 445 (7123): 111–15. <a href="https://doi.org/10.1038/nature05384">https://doi.org/10.1038/nature05384</a>.
</div>
<div id="ref-rich2007" class="csl-entry" role="listitem">
Rich, Jeremy N. 2007. <span>“Cancer Stem Cells in Radiation Resistance.”</span> <em>Cancer Research</em> 67 (19): 8980–84. <a href="https://doi.org/10.1158/0008-5472.CAN-07-0895">https://doi.org/10.1158/0008-5472.CAN-07-0895</a>.
</div>
<div id="ref-schonberg2014" class="csl-entry" role="listitem">
Schonberg, David L., Daniel Lubelski, Tyler E. Miller, and Jeremy N. Rich. 2014. <span>“Brain Tumor Stem Cells: Molecular Characteristics and Their Impact on Therapy.”</span> <em>Molecular Aspects of Medicine</em> 39 (October): 82–101. <a href="https://doi.org/10.1016/j.mam.2013.06.004">https://doi.org/10.1016/j.mam.2013.06.004</a>.
</div>
<div id="ref-sell1993" class="csl-entry" role="listitem">
Sell, S. 1993. <span>“Cellular Origin of Cancer: Dedifferentiation or Stem Cell Maturation Arrest?”</span> <em>Environmental Health Perspectives</em> 101 (suppl 5): 15–26. <a href="https://doi.org/10.1289/ehp.93101s515">https://doi.org/10.1289/ehp.93101s515</a>.
</div>
<div id="ref-singh2003" class="csl-entry" role="listitem">
Singh, Sheila K., Ian D. Clarke, Mizuhiko Terasaki, Victoria E. Bonn, Cynthia Hawkins, Jeremy Squire, and Peter B. Dirks. 2003. <span>“Identification of a Cancer Stem Cell in Human Brain Tumors.”</span> <em>Cancer Research</em> 63 (18): 5821–28.
</div>
<div id="ref-singh2004" class="csl-entry" role="listitem">
Singh, Sheila K., Cynthia Hawkins, Ian D. Clarke, Jeremy A. Squire, Jane Bayani, Takuichiro Hide, R. Mark Henkelman, Michael D. Cusimano, and Peter B. Dirks. 2004. <span>“Identification of Human Brain Tumour Initiating Cells.”</span> <em>Nature</em> 432 (7015): 396–401. <a href="https://doi.org/10.1038/nature03128">https://doi.org/10.1038/nature03128</a>.
</div>
<div id="ref-sweeney1995" class="csl-entry" role="listitem">
Sweeney, Eamon. 1995. <span>“Dormant Cells in Columnar Cell Carcinoma of the Thyroid.”</span> <em>Human Pathology</em> 26 (6): 691–92. <a href="https://doi.org/10.1016/0046-8177(95)90180-9">https://doi.org/10.1016/0046-8177(95)90180-9</a>.
</div>
<div id="ref-taipale2001" class="csl-entry" role="listitem">
Taipale, Jussi, and Philip A. Beachy. 2001. <span>“The Hedgehog and Wnt Signalling Pathways in Cancer.”</span> <em>Nature</em> 411 (6835): 349–54. <a href="https://doi.org/10.1038/35077219">https://doi.org/10.1038/35077219</a>.
</div>
<div id="ref-tang2021" class="csl-entry" role="listitem">
Tang, Xuejia, Chenghai Zuo, Pengchao Fang, Guojing Liu, Yongyi Qiu, Yi Huang, and Rongrui Tang. 2021a. <span>“Targeting Glioblastoma Stem Cells: A Review on Biomarkers, Signal Pathways and Targeted Therapy.”</span> <em>Frontiers in Oncology</em> 11 (July): 701291. <a href="https://doi.org/10.3389/fonc.2021.701291">https://doi.org/10.3389/fonc.2021.701291</a>.
</div>
<div id="ref-tang2021a" class="csl-entry" role="listitem">
———. 2021b. <span>“Targeting Glioblastoma Stem Cells: A Review on Biomarkers, Signal Pathways and Targeted Therapy.”</span> <em>Frontiers in Oncology</em> 11 (July): 701291. <a href="https://doi.org/10.3389/fonc.2021.701291">https://doi.org/10.3389/fonc.2021.701291</a>.
</div>
<div id="ref-turner2009" class="csl-entry" role="listitem">
Turner, C., A. R. Stinchcombe, M. Kohandel, S. Singh, and S. Sivaloganathan. 2009. <span>“Characterization of Brain Cancer Stem Cells: A Mathematical Approach.”</span> <em>Cell Proliferation</em> 42 (4): 529–40. <a href="https://doi.org/10.1111/j.1365-2184.2009.00619.x">https://doi.org/10.1111/j.1365-2184.2009.00619.x</a>.
</div>
<div id="ref-vlashi2009" class="csl-entry" role="listitem">
Vlashi, Erina, Kwanghee Kim, Chann Lagadec, Lorenza Della Donna, John Tyson McDonald, Mansoureh Eghbali, James W. Sayre, Encrico Stefani, William McBride, and Frank Pajonk. 2009. <span>“In Vivo Imaging, Tracking, and Targeting of Cancer Stem Cells.”</span> <em>JNCI: Journal of the National Cancer Institute</em> 101 (5): 350–59. <a href="https://doi.org/10.1093/jnci/djn509">https://doi.org/10.1093/jnci/djn509</a>.
</div>
<div id="ref-weiss2017a" class="csl-entry" role="listitem">
Weiss, Lora D., Natalia L. Komarova, and Ignacio A. Rodriguez-Brenes. 2017. <span>“Mathematical Modeling of Normal and Cancer Stem Cells.”</span> <em>Current Stem Cell Reports</em> 3 (3): 232–39. <a href="https://doi.org/10.1007/s40778-017-0094-4">https://doi.org/10.1007/s40778-017-0094-4</a>.
</div>
<div id="ref-yan2016" class="csl-entry" role="listitem">
Yan, Min, and Quentin Liu. 2016. <span>“Differentiation Therapy: A Promising Strategy for Cancer Treatment.”</span> <em>Chinese Journal of Cancer</em> 35 (1): 3. <a href="https://doi.org/10.1186/s40880-015-0059-x">https://doi.org/10.1186/s40880-015-0059-x</a>.
</div>
<div id="ref-youssefpour2012" class="csl-entry" role="listitem">
Youssefpour, H., X. Li, A. D. Lander, and J. S. Lowengrub. 2012. <span>“Multispecies Model of Cell Lineages and Feedback Control in Solid Tumors.”</span> <em>Journal of Theoretical Biology</em> 304 (July): 39–59. <a href="https://doi.org/10.1016/j.jtbi.2012.02.030">https://doi.org/10.1016/j.jtbi.2012.02.030</a>.
</div>
</div>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./summary.html" class="pagination-link" aria-label="Summary">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Summary</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./preprint.html" class="pagination-link" aria-label="Virtual Clinical Trials of BMP4 Differentiation Therapy: Digital Twins to Aid Glioblastoma Trial Design">
        <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Virtual Clinical Trials of BMP4 Differentiation Therapy: Digital Twins to Aid Glioblastoma Trial Design</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>Nicholas Harbour Second year progress review</p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    <div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/Harbour-N/Second-year-report/edit/main/Lit_review.qmd" class="toc-action"><i class="bi bi-github"></i>Edit this page</a></li><li><a href="https://github.com/Harbour-N/Second-year-report/issues/new" class="toc-action"><i class="bi empty"></i>Report an issue</a></li></ul></div></div>
    <div class="nav-footer-right">
<p>This book was built with <a href="https://quarto.org/">Quarto</a>.</p>
</div>
  </div>
</footer>




</body></html>